app/src/app/content-plugin/components/header/header.component.ts

```diff
@@ -55,14 +62,14 @@ export class AppHeaderComponent implements OnInit, OnDestroy {
 
   actions: Array<ContentActionRef> = [];
 
-  constructor(store: Store<AppStore>, private appExtensions: AppExtensionService, private appConfigService: AppConfigService) {
+  constructor(public store: Store<AppStore>, private appExtensions: AppExtensionService, private appConfigService: AppConfigService) {
     this.headerColor$ = store.select(getHeaderColor);
     this.headerTextColor$ = store.select(getHeaderTextColor);
     this.appName$ = store.select(getAppName);
     this.logo$ = store.select(getLogoPath);
     this.landingPage = this.appConfigService.get('landingPage', '/personal-files');
 
-    store.select(getHeaderImagePath).subscribe((path) => {
+    this.store.select(getHeaderImagePath).subscribe((path) => {
```

Comment:
I think you don't need `this` even if the `store` is public

Reply:
reverted